weaver/core/relay/Cargo.toml

```diff
@@ -37,8 +37,8 @@ futures = { version = "0.3.27" }
 base64 = "0.20.0"
 reqwest = { version = "0.11.16", features = ["json"] }
 serde_json = "1.0.108"
-cacti_weaver_protos_rs = { path = "./protos-rs" }
-# cacti_weaver_protos_rs = "2.0.0-alpha.2"
+cacti_weaver_protos_rs = "2.0.0-rc.1"
```

Comment:
This lines needs to be reversed, it was pointing to local `protos-rs` module.
Also by default the line with version of `protos-rs` i.e. line 41 should be uncommented, and line 40 (one with path to local `protos-rs`) should be commented. I'm not sure how it became the other way, can you please make this change?

Reply:
@sandeepnRES I've updated it so that the local version is being used, thank you for the review!